compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -15108,7 +15112,7 @@ private STNode parseTypeDescOrExpr(List<STNode> qualifiers) {
                 return parseTypeDescOrExprRhs(typeOrExpr);
             case OPEN_BRACKET_TOKEN:
                 reportInvalidQualifierList(qualifiers);
-                typeOrExpr = parseTupleTypeDescOrExprStartsWithOpenBracket();
+                typeOrExpr = parseTupleTypeDescOrListConstructor(STNodeFactory.createEmptyNode());
```

Comment:
```suggestion
                typeOrExpr = parseTupleTypeDescOrListConstructor(STNodeFactory.createEmptyNodeList());
```
the other place also.

Reply:
Updated thanx